pyupgrade/_plugins/versioned_branches.py

```diff
@@ -98,6 +106,11 @@ def visit_If(
         node: ast.If,
         parent: ast.AST,
 ) -> Iterable[Tuple[Offset, TokenFunc]]:
+    if state.settings.min_version >= (3,) and (
+        len(state.settings.min_version) >= 2
+    ):
+        py3_minor = state.settings.min_version[1]
```

Comment:
this makes a sometimes-present variable which is a bit of a hazard

the whole patch can be very much simplified by changing `(3,)` => `(3, 0)` and then you don't need any special logic

Reply:
> this makes a sometimes-present variable which is a bit of a hazard

point taken.

> the whole patch can be very much simplified by changing (3,) => (3, 0) and then you don't need any special logic

Not so sure about that: `state.settings.min_version` isn't guaranteed to have a second item (with `--py3-plus` it's just `(3,)`), but `(3,) >= (3, 0)` evaluates to `False`. Should I change this ?

More generally I _think_ you're right that it can be simplified, at least it feels a lot like it, I just don't see how right now.

Reply:
I think you can just set:
```python
if state.settings.min_version == (3,):
    min_version = (3, 0)
else:
    min_version = state.settings.min_version
```

or something of the sort